tools_galaxyp.yaml

```diff
@@ -210,4 +210,8 @@ tools:
     
   - name: metaquantome_viz
     owner: galaxyp
-    tool_panel_section_label: Proteomics  
+    tool_panel_section_label: Proteomics 
+    
+  - name: maxbin2
+    owner: mbernt
```

Comment:
@mbernt can we move this to IUC or GalaxyP?

We would need to track yet another repo for creating containers and so on ...

Reply:
This is now in IUC ... thanks @bernt-matthias